__tests__/index.test.js

```diff
@@ -124,97 +124,129 @@ describe("GET /api/articles", () => {
 
 describe("GET /api/articles/:article_id/comments", () => {
   test("status 200: responds with an array of comment objects", () => {
-      return request(app)
+    return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
       .then(({ body }) => {
-              expect(body.length).toBe(11);
-              expect(body[0].article_id).toBe(1);
-          body.forEach((comment) => {
-              expect(typeof comment.comment_id).toBe("number");
-              expect(typeof comment.votes).toBe("number");
-              expect(typeof comment.created_at).toBe("string");
-              expect(typeof comment.author).toBe("string");
-              expect(typeof comment.body).toBe("string");
-          });
+        expect(body.length).toBe(11);
+        expect(body[0].article_id).toBe(1);
+        body.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+        });
       });
   });
 
   test("status 200: responds with an array of comment objects sorted by date in descending order.", () => {
     return request(app)
-    .get("/api/articles/1/comments")
-    .expect(200)
-    .then(({ body }) => {
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
         expect(body).toBeSortedBy("created_at", { descending: true });
-    });
-});
+      });
+  });
 
   test("status 200: responds with an empty array when given an article_id with no comments", () => {
-      return request(app)
+    return request(app)
       .get("/api/articles/2/comments")
       .expect(200)
       .then(({ body }) => {
         expect(body.msg).toBe("Comments not found");
```

Comment:
You've said it responds with an empty array in your description, but then you're checking for a message?